src/main/java/com/easyrest/components/TimePicker.java

```diff
@@ -0,0 +1,77 @@
+package com.easyrest.components;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+import java.util.List;
+
+public class TimePicker {
+
+    private WebDriver driver;
+
+    private final By amButton = By.xpath("//*[text()='AM']");
+    private final By pmButton = By.xpath("//*[text()='PM']");
+    private final By hour = By.xpath("//h2[contains (@class, 'MuiPickersToolbarButton')][1]");
+    private final By minute = By.xpath("//h2[contains (@class, 'MuiPickersToolbarButton')][3]");
+    private final By hoursInClock = By.xpath("//span[contains(@class,'MuiPickersClockNumber-clockNumber')]");
+    private final By minutesInClock = By.xpath("//span[contains(@class,'MuiPickersClockNumber-clockNumber')]");
+    WebElement clockPointer = driver.findElement(By.xpath("//div[contains(@class,'MuiPickersClockPointer-thumb')]"));
```

Comment:
Are you sure that the default access modifier will be working here, maybe private would be better?

Reply:
agree, done